.phpstorm.meta.php

```diff
@@ -5,13 +5,17 @@
 // see https://confluence.jetbrains.com/display/PhpStorm/PhpStorm+Advanced+Metadata
 namespace PHPSTORM_META;
 
+use Psr\Container\ContainerInterface;
+use PHPStan\PhpDocParser\Ast\Node;
+use PHPStan\PhpDocParser\Ast\NodeAttributes;
+use Rector\NodeTypeResolver\Node\AttributeKey;
 use Rector\BetterPhpDocParser\ValueObject\PhpDocAttributeKey;
 
 // $container->get(Type::class) → instance of "Type"
-override(\Psr\Container\ContainerInterface::get(0), type(0));
+override((new ContainerInterface())->get(0), type(0));
```

Comment:
Could you split these meta changes to its own PR? Just so we have changes independent in case of blame/regression

Reply:
I think, that without changing this file, there would be a failed CI for the current PR.

Here's the separate PR https://github.com/rectorphp/rector-src/pull/6593